src/network/graphMutation.js

```diff
@@ -129,8 +129,9 @@ export function createFinalizeResultTx(version, topicAddress, oracleAddress, sen
   return new GraphMutation('finalizeResult', args, TYPE.transaction).execute();
 }
 
-export function createWithdrawTx(version, topicAddress, senderAddress) {
+export function createWithdrawTx(type, version, topicAddress, senderAddress) {
   const args = {
+    type,
```

Comment:
you know you can get all the args like this?
```
export const createWithdrawTx = (...args) => new GraphMutation('withdraw', args, TYPE.transaction).execute();
```

Reply:
jk, they're an array instead of an object then.

Reply:
lol